src/Jackalope/Transport/Jackrabbit/Request.php

```diff
@@ -463,6 +477,10 @@ protected function singleRequest($getCurlObject)
         $curl->setopt(CURLOPT_URL, reset($this->uri));
         $curl->setopt(CURLOPT_HTTPHEADER, $headers);
         $curl->setopt(CURLOPT_POSTFIELDS, $this->body);
+
+        if (true === $this->forceHttpVersion10) {
+            $curl->setopt(CURLOPT_HTTP_VERSION, CURL_HTTP_VERSION_1_0);
```

Comment:
I was going to add this to the `multiRequest` method, however I noticed that it is not called at anypoint during the test suite.